topics/imaging/tutorials/tutorial-CP/slides.html

```diff
@@ -0,0 +1,123 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: Nucleoli segmentation <br> & <br> feature extraction <br> using CellProfiler
+zenodo_link: ''
+questions:
+- How do I run an image analysis pipeline on public data using CellProfiler?
+- How do I analyse the DNA channel of fluorescence siRNA screens?
+- How do I download public image data into my history?
+- How do I segment and label cell nuclei?
+- How do I segment nucleoli (as the absence of DNA)?
+- How do I combine nuclei and nucleoli into one segmentation mask?
+- How do I extract the background of an image?
+- How do I relate the nucleoli to their parent nucleus?
+- How do I measure the image and object features?
+- How do I measure the image quality?
+objectives:
+- How to download images from a public image repository.
+- How to segment cell nuclei using CellProfiler in Galaxy.
+- How to segment cell nucleoli using CellProfiler in Galaxy.
+- How to extract features for images, nuclei and nucleoli.
+time_estimation: 4H
+key_points:
+- Galaxy workflows can download images from the IDR, selecting specific channels, time points, z-stack positions and crop the image in different ways.
+- CellProfiler in Galaxy can segment and extract features of any object of interest.
+- The features and masks can be exported for further analysis.
+contributors:
+- beatrizserrano
+---
+
+
+### What is the nucleoli?
```

Comment:
you could add comments for the presenter to your slides if you like

Reply:
done